### PR TITLE
Support subdirectory and `--enable_warnings`

### PR DESCRIPTION
**Pull request by Defined2014**

### CODE REVIEW

File: src/main.go [hunk 1]
```diff
@@ -251,11 +252,13 @@ func (t *tester) preProcess() {
 
 	log.Warn("Create new db", mdb)
 
-	if _, err = mdb.Exec(fmt.Sprintf("create database `%s`", t.name)); err != nil {
-		log.Fatalf("Executing create db %s err[%v]", t.name, err)
+	dbName = t.name
+	dbName = strings.ReplaceAll(dbName, "/", "__")
```
Comment (bb7133) [inline src/main.go:256]: Just combine them into 1 line?

`dbName = strings.ReplaceAll(t.name, "/", "__")`

File: src/main.go [hunk 2]
```diff
@@ -274,7 +277,9 @@ func (t *tester) preProcess() {
 
 func (t *tester) postProcess() {
 	if !reserveSchema {
-		t.mdb.Exec(fmt.Sprintf("drop database `%s`", t.name))
+		dbName := t.name
```
Comment (bb7133) [inline src/main.go:280]: Ditto

File: src/main.go [hunk 3]
```diff
@@ -605,7 +610,7 @@ func (t *tester) parserErrorHandle(query query, err error) error {
 	}
 	err = syntaxError(err)
 	for _, expectedErr := range t.expectedErrs {
-		if expectedErr == "ER_PARSE_ERROR" {
+		if expectedErr == "ER_PARSE_ERROR" || expectedErr == "1064" {
```
Comment (bb7133) [inline src/main.go:613]: What does '1064' mean?

Comment (Defined2014) [reply src/main.go:610]: The error code, like example below
```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near
```

File: src/main.go (hunk 1 above)
Comment (Defined2014) [reply src/main.go]: Upadted

File: src/main.go (hunk 2 above)
Comment (Defined2014) [reply src/main.go]: Updated

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**